### ISSUE TITLE
"Something went wrong" error message if we give the wrong "Activation email"

### ISSUE THREAD

**Issue by laranz** [AyeCode/userswp#522]
![image](https://user-images.githubusercontent.com/208945/117699741-ce7dac80-b1e2-11eb-9202-3a77b74e25e8.png)

Can we give a proper error message? Instead "Something went wrong" or give a link to send another activation mail there?

**Comment by laranz** [comment]
@wpdev10 

1. I Created a user with the "Email Activation" options.
2. I am using "Email Log" plugin to get emails in the Backend.
3. I get the activation link.
4. Now in the frontend, there is an option to get another activation email after we successfully registered.
5. I click that.
6. Now I got two activation mail. ( Only the latest works )
7. I copy the old activation link, and paste it in the address bar, and get this error.
